lib/src/components/NxSidebarNavigation/NxSidebarNavitation.scss

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-width__open: 250px;
+$nx-sidebar-width__closed: 69px;
+$nx-sidebar-logo-height: 33px;
+$nx-sidebar-nav-background-color: $nx-blue-1000;
+$nx-sidebar-nav-text-color: $nx-white;
+$nx-sidebar-nav-help-color: $nx-grey-50;
+$nx-sidebar-nav-hover-color: $nx-indigo-600;
+$nx-sidebar-nav-selected-color: $nx-indigo-800;
+$nx-sidebar-separator-color: $nx-grey-900;
+
+
+.nx-sidebar-nav {
+  @include container-vertical;
+
+  box-sizing: border-box;
+  background-color: $nx-sidebar-nav-background-color;
+  color: $nx-sidebar-nav-text-color;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  min-width: 250px;
+  padding: 0;
+  padding-top: $nx-spacing-l;
+  position: relative;
+  // transition: all 0.5s ease;
+}
+
+.nx-sidebar-nav--product-info {
+  margin: $nx-spacing-sm $nx-spacing-l 0;
+  height: $nx-sidebar-logo-height;
+}
+
+.nx-sidebar-nav--product-name {
+  display: inline-block;
+  font-size: 16px;
+  line-height: $nx-sidebar-logo-height;
+  vertical-align: top;
+}
+
+.nx-sidebar-nav--logo {
+  height: $nx-sidebar-logo-height;
+  margin-right: $nx-spacing-xs;
+  width: 30px;
+}
+
+.nx-sidebar-nav--icon {
+  color: $nx-sidebar-nav-text-color;
+  margin: 0;
+}
+
+.nx-footer-help-icon {
+  color: $nx-sidebar-nav-help-color;
+}
+
+.nx-sidebar-nav--footer {
+  align-self: center;
+  bottom: $nx-spacing-xxl;
+  font-size: $nx-font-size-xs;
+  position: absolute;
+  text-align: center;
+}
+
+.nx-sidebar-nav--items {
+  list-style-type: none;
+  margin: 36px 0;
+  padding: 0;
+  width: 100%;
+}
+
+.nx-sidebar-nav--item {
+  height: 32px;
+
+  &:hover {
+    background-color: $nx-sidebar-nav-hover-color;
+  }
+
+  &.selected {
+    background-color: $nx-sidebar-nav-selected-color;
+
+    .nx-sidebar-nav--link {
+      @include semi-bold;
+    }
+  }
+}
+
+.nx-sidebar-nav--separator {
+  background-color: $nx-sidebar-separator-color;
+  height: 1px;
+  width: 100%;
+}
+
+.nx-sidebar-nav--release {
+  height: 36px;
+  margin-bottom: $nx-spacing-sm;
+  margin-top: $nx-spacing-xxl;
+}
+
+.nx-sidebar-nav--link {
```

Comment:
Long text problems:
![image](https://user-images.githubusercontent.com/3630091/108731532-a7281500-74fa-11eb-9590-97c142fbe621.png)

Reply:
> Long text problems:
> ![image](https://user-images.githubusercontent.com/3630091/108731532-a7281500-74fa-11eb-9590-97c142fbe621.png)

Spoke about this today at wall walk and we (@stephenim and I) believe this to be irrelevant because we at Sonatype —and specifically the design org— controls the copy that will go into these links.

Reply:
> Spoke about this today at wall walk and we (@stephenim and I) believe this to be irrelevant because we at Sonatype —and specifically the design org— controls the copy that will go into these links.

Sounds good.  Remember to specifically mention that limitation when writing the docs for this component.